src/track/beats.cpp

```diff
@@ -24,6 +26,7 @@ constexpr double kEpsilon = 0.01;
 } // namespace
 
 namespace mixxx {
+using std::transform;
```

Comment:
lets avoid `using` at namespace scope. Especially with a namespace this large. I don't see how this is necessary anyways.